resources/assets/js/videos/components/thumbnailPreview.vue

```diff
@@ -266,6 +266,14 @@ export default {
         }
     },
     watch: {
+        duration() {
+            this.setSpritesFolderpath();
+            this.updateThumbnailInterval();
+
+            this.spriteIdx = Math.floor(this.hoverTime / (this.thumbnailInterval * this.thumbnailsPerSprite));
+            this.updateSprite();
+            this.viewHoverTimeBar();
+        },
```

Comment:
Why is this needed? With the new implementation the duration should never change, except at first from 0 to >0. The thumbnail preview is only shown once the duration is >0.

Reply:
Can you explain why the duration should never change? Every video has a different duration. Or did you mean something different?

Reply:
I added the watcher because otherwise the thumbnails would no longer be updated. Previously, when the video changed, the duration was reset to 0, which caused the thumbnail component to be destroyed and recreated. This ensured that the data was always freshly loaded. Now, the duration is set to 0 only once at the beginning, so the thumbnails need to be updated when the video changes.

Reply:
Makes sense, thanks. I didn't consider multiple videos in the same volume :facepalm: